src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -873,6 +875,24 @@ public void CanResize(bool value)
             UpdateWindowProperties(newWindowProperties);
         }
 
+        public void SetCanMinimize(bool value)
+        {
+            var newWindowProperties = _windowProperties;
+
+            newWindowProperties.IsMinimizable = value;
+
+            UpdateWindowProperties(newWindowProperties);
```

Comment:
Just a question. Before this PR, developers were likely to call PInvokes directly to set these properties to hide minimize. Would it make sense to read initial values of _windowProperties from the API? It might be tricky, as we don't know at which point these properties are set.

Reply:
Probably not worth it.

Reply:
This PR shouldn't break anyone that was removing `WS_MINIMIZEBOX` using PInvoke or `Win32Properties.WindowStylesCallback`. The logic hasn't changed at all when `CanMinimize = true`.